### PR TITLE
misc cleanup

### PR DESCRIPTION
**Pull request by nickpell**
Might've caught https://github.com/dimagi/commcare-hq/pull/11369 earlier if this file didn't clutter my IDE with warnings.  This helps clean out most of them.

🐟 or 🏣 

@esoergel

### CODE REVIEW

**Comment by nickpell** [comment]
✅

File: corehq/apps/accounting/models.py [hunk 1]
```diff
@@ -1049,13 +1053,13 @@ def __str__(self):
...
-            and other.date_end == self.date_end
-            and other.subscriber.pk == self.subscriber.pk
-            and other.account.pk == self.account.pk
+            other is not None and
+            other.__class__.__name__ == self.__class__.__name__ and
+            other.plan_version.pk == self.plan_version.pk and
+            other.date_start == self.date_start and
+            other.date_end == self.date_end and
+            other.subscriber.pk == self.subscriber.pk and
+            other.account.pk == self.account.pk
```
Comment (esoergel) [inline corehq/apps/accounting/models.py]: This one I'm not sure of - it seems to be the convention in our codebase to put the line break *before* the binary operator.

**Comment by esoergel** [comment]
Big fan!

File: corehq/apps/accounting/models.py (hunk 1 above)
Comment (nickpell) [inline corehq/apps/accounting/models.py]: removed these changes

**Comment by nickpell** [comment]
should be good to merge

**Pull request merged**